docs/Prefill-decoding_Disaggregation.md

```diff
@@ -0,0 +1,25 @@
+# Prefill-decoding Disaggregation
+Existing large language model (LLM) serving systems that support prefill-decoding disaggregation typically compute the prefill and decoding phases on separate instances. For each request, following the prefill phase, the system migrates the generated key-value (KV) cache to the decoding instance and continues the computation. 
+
+
+Llumnix can manage the scheduling of runtime requests across instances. It can perceive the execution process of requests and the state of the KV cache. Llumnix provides a series of scheduling semantics to support the initial dispatch and runtime rescheduling (request migration across instances). 
+
+
+Therefore, Llumnix is inherently well-suited for the "special" cross-instance scheduling requirements in prefill-decoding disaggregation. Specifically, it limits the initial **dispatch only to the prefill instances** and the **migration between prefill and decoding instances**. 
+
+## Supported Features
+1. Requests can be **automatically migrated** from prefill instance to decoding instances.
+
+2. Users can specify the number of prefill instances.
+
+3. Llumnix supports both **one-to-many and many-to-one migrations** from prefill to decoding instances.
```

Comment:
no need to mention "one-to-many and many-to-one". That's only implementation details